README.md

```diff
@@ -117,33 +117,7 @@ To run individual test suites, use `make eunit t=SUITE` where SUITE is one of
 The encode and protocol test suites does not require a
 running MySQL server on localhost.
 
-For the suites `mysql_tests`, `ssl_tests` and `transaction_tests` you need to
-start MySQL on localhost and give privileges to the users `otptest`, `otptest2`
-and (for `ssl_tests`) to the user `otptestssl`:
-
-```SQL
-CREATE USER otptest@localhost IDENTIFIED BY 'OtpTest--123';
-GRANT ALL PRIVILEGES ON otptest.* TO otptest@localhost;
-
-CREATE USER otptest2@localhost IDENTIFIED BY 'OtpTest2--123';
-GRANT ALL PRIVILEGES ON otptest.* TO otptest2@localhost;
-
--- in MySQL < 5.7, REQUIRE SSL must be given in GRANT
-CREATE USER otptestssl@localhost IDENTIFIED BY 'OtpTestSSL--123';
-GRANT ALL PRIVILEGES ON otptest.* TO otptestssl@localhost REQUIRE SSL;
-
--- in MySQL >= 8.0, REQUIRE SSL must be given in CREATE USER
-CREATE USER otptestssl@localhost IDENTIFIED BY 'OtpTestSSL--123' REQUIRE SSL;
-GRANT ALL PRIVILEGES ON otptest.* TO otptestssl@localhost;
-```
-
-Before running the test suite `ssl_tests` you'll also need to generate SSL files
-and MySQL extra config file. In order to do so, please execute `make tests-prep`.
-
-The MySQL server configuration must include `my-ssl.cnf` file,
-which can be found in `test/ssl/`.
-**Do not run** `make tests-prep` after you start MySQL,
-because CA certificates will no longer match.
+To quickly setup a test environment, execute `.ci/prep.sh`, then execute `make tests`.
```

Comment:
If the `.ci/` stuff is used also for local tests, then maybe we should name it something other than CI? And not a hidden directory? Something under `test` such as `test/setup/` or `test/util/` or something like that, or is it not good to put things under test? (We already have `test/ssl/`. (If you don't like to put scripts under test/, then we can move the ssl cert scripts too, to keep all in one place.)

Do you think nobody should run the tests without docker from now on? I'm not sure we should require running a script that needs sudo.

The scripts are already divided into the ones that are used for docker and the ones that are used also when running without docker. For example `.ci/init.sh` is the script that creates the SQL users and `make test-prep` sets up the certificates. We can delete the SQL code but maybe keep some of the explanations about which steps are required and what each of the scripts do.

Reply:
> If the .ci/ stuff is used also for local tests, then maybe we should name it something other than CI? 

Yes, good point. I can do this in a follow up PR.
The priority of this PR is to get GitHub Actions up and running to make future PRs verified.

> Do you think nobody should run the tests without docker from now on? I'm not sure we should require running a script that needs sudo.

Everyone wants a quick feedback loop should use the scripts provided in this repo to setup MySQL.
The unix domain socket tests will not work pass otherwise.

Reply:
I think it can be refactored a bit to separate what needs sudo, what runs with docker, what doesn't.

As of now, I don't want to delete the README instructions about setting up running the tests locally without docker though, so please undelete those parts, but we can keep the link to the `.ci` script too.

Reply:
In the latest commit, I added instructions about how to run tests against MySQL or MariaDB running in localhost.

I am not convinced we should keep the SQL statements in README because it's hard to keep it up-to-date.
It should be enough to reference to init.sh from README